src/BPEssentials/Commands/Home/Home.cs

```diff
@@ -0,0 +1,50 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BrokeProtocol.Entities;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace BPEssentials.Commands
+{
+    public class Home : Command
+    {
+        public void Invoke(ShPlayer player, int homeNumber)
```

Comment:
```suggestion
        public void Invoke(ShPlayer player, int homeNumber = 0)
```
Make it default to 0.

Reply:
Oh I will make it default to 1 (I made it that homes start at 1 and not 0